src/sections/navbar.html

```diff
@@ -0,0 +1,31 @@
+<div class="flex justify-between bg-PrimaryColor p-8 sticky w-full top-0 z-10">
+  <div class="flex flex-col text-smallerSize md:text-2xl">
+    <h1>DE</h1>
+    <h1>EN</h1>
```

Comment:
here, the text applies a bit bigger because they don't have a font weight, try using font-light